src/sentry/testutils/helpers/task_runner.py

```diff
@@ -26,25 +26,25 @@ def BurstTaskRunner():
     worker.
     """
 
-    queue = []
+    job_queue = []
 
-    def apply_async(self, args=(), kwargs=(), countdown=None):
-        queue.append((self, args, kwargs))
+    def apply_async(self, args=(), kwargs=(), countdown=None, queue=None):
```

Comment:
this is so our monkeypatch can use the queue parameter that didn't used to get passed to it